### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by noelfoka**
## Math magicians: full website.

In this branch, I:

- [ ] Create a website consisting of 3 pages: `Home`, `Calculator`, and `Quote`.
- [ ] Add my own styles to improve the look and feel.
- [ ] For every page, I should create a `<Route>`.
- [ ] Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/routes-component/Home.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        until 1970, when a company in Japan invented the first digital pocket
+        calculator and the demand for the antiquated device faded. Companies in
+        the United States, such as Texas Instruments, adapted the design of the
+        Japanese device, and enhanced it by creating the graphing calculators we
+        know today. With the invention of the first smartphone in 1995,
+        individuals began to replace expensive digital calculators with the
+        multiuse device. This required even the most sophisticated calculator
+        designs to be upgraded in order to remain relevant in the market.
+      </p>
+    </div>
```
Comment (danmainah) [inline src/routes-component/Home.js:52]: - [ ] @noelfoka Great work on your homepage however the layout doesn't match the design given, kindly check the comparison below for a better understanding:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
image  |  image

File: src/components/Calculator.js [hunk 2]
```diff
@@ -33,96 +28,77 @@ class Calculator extends Component {
...
+          onClick={clickEvent}
+          data-testid="answer"
         >
           =
         </button>
       </div>
-    );
-  }
-}
+    </div>
```
Comment (danmainah) [inline src/components/Calculator.js:100]: - [ ] @noelfoka Great work on your calculator page, however there is a small error in placement of the title, check the comparison below for better understanding:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @noelfoka , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Added all three pages home, calculator, and quote
✔️ Used [React Router V6](https://reactrouter.com/). for navigation between the pages

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] It would be nice if you correct the error in the getting started section we should use `npm start`  and not `npm run build` to run the project after cloning it.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by KDlamini — APPROVED** [review]
### Hi @noelfoka 🙋‍♀️, 

Excellent work! 💯 
You have done a great job creating a beautiful Math magician's full website. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!


## Highlights 🌞 

✔️  Professional pull request
✔️  Well documented README
✔️  The website consists of 3 pages: `Home`, `Calculator`, and `Quote`
✔️  Pages have a **dedicated route** to it made possible by [React Router V6](https://reactrouter.com/)

# Status: Approved ✔️ 

oscars-standing-ovation
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**